Mixpanel/MixpanelPrivate.h

```diff
@@ -112,6 +112,7 @@
 @property (nonatomic) BOOL hasAddedObserver;
 @property (nonatomic, strong) NSNumber *automaticEventsEnabled;
 @property (nonatomic, strong) NSArray *notifications;
+@property (nonatomic, strong) NSArray *triggeredNotifications;
```

Comment:
using (nonatomic, copy) is recommended, since you are here, could you fix it for notifications to @property (nonatomic, copy) as well? In objc, use copy for classes that are part of a class cluster that have mutable/immutable pairs; such as NSString/NSMutableString NSArray/NSMutableArray NSDictionary/NSMutableDictionary NSSet/NSMutableSet